pkg/cmd/workspace/util/creation_data.go

```diff
@@ -15,13 +15,21 @@ import (
 	"github.com/daytonaio/daytona/pkg/views/workspace/create"
 )
 
+type CmdCustoms struct {
```

Comment:
I suggest we call this `Defaults` and remove `ApiServerConfig`. Then the calling function could decide to set defaults based on provided flags or api server config values.